src/build.ps1

```diff
@@ -0,0 +1,16 @@
+$ErrorActionPreference = "Stop"
```

Comment:
Would it be possible to use build.cmd instead? Or additionally? I have couple of machines where Powershell is disabled for some reason.

Reply:
Sure, no problem.